frappe/translations/ru.csv

```diff
@@ -914,7 +914,7 @@ Document can't saved.,Документ не может быть сохранен
 Document {0} has been set to state {1} by {2},Документ {0} установлен в состояние {1} на {2},
 Documents,Документы,
 Documents assigned to you and by you.,"Документы, назначенные вам и вами.",
-Domain Settings,Настройки домена,
+Domain Settings,Настройка сфер деятельности,
```

Comment:
the previous definition seems correct according to google?

before:

![Screenshot 2022-06-01 at 1 09 54 PM](https://user-images.githubusercontent.com/32034600/171352913-8b70f536-7a9d-43ac-ab9e-74c46e983205.png)

after:

![Screenshot 2022-06-01 at 1 09 36 PM](https://user-images.githubusercontent.com/32034600/171352981-8865cf01-b879-448d-930d-69a27ee70675.png)